airflow/dags/metrcis/load_firestore_to_snowflake.py

```diff
@@ -0,0 +1,89 @@
+"""
+This DAG is responsible for pulling requests from Firestore and writing them
+to snowflake.
+"""
+
+import os
+import time
+from datetime import date, datetime, timedelta
+
+import snowflake.connector
+from google.cloud import firestore
+from google.cloud.firestore_v1.base_query import FieldFilter
+from include.utils.slack import send_failure_notification
+
+from airflow.decorators import dag, task
+
+METRICS_SNOWFLAKE_DB_USER = os.environ.get("METRICS_SNOWFLAKE_DB_USER")
```

Comment:
Should we not get these from the snowflake connection of airflow instead?

Reply:
We could, but I'm not sure whether we want. We'll need to install the provider simply to get the conn.

Reply:
Could we not use this operator to run the query: https://airflow.apache.org/docs/apache-airflow-providers-snowflake/stable/operators/snowflake.html#snowflakesqlapioperator?

Reply:
But I am okay with current implementation for now but we could track this in future PRs.